bpf/rtt_tracker.h

```diff
@@ -89,12 +27,12 @@ static inline int rtt_lookup_and_update_flow(flow_id *id, u16 flags, u64 rtt) {
 }
 
 static inline int calculate_flow_rtt_tcp(struct sock *sk, struct sk_buff *skb) {
+    u8 dscp = 0, protocol = 0;
     struct tcp_sock *ts;
     u16 family, flags = 0;
```

Comment:
could you initialize family to 0?
It's later passed down into `core_fill_in_l2` which does the initialization but not in all paths

Reply:
its initialized at line 32

Reply:
flags yes, but family no: `u16 family, flags = 0;`